app/book.py

```diff
@@ -0,0 +1,12 @@
+from dataclasses import dataclass
+
+
+@dataclass
+class Book:
+    title: str
+    content: str
```

Comment:
why not usual python classes?)

Reply:
Is it bad practice to use dataclasses?

Reply:
@msymonovych just curious why not using usual classes for this task :) And no, it isn't, but it real projects usual classes are used more often